scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -879,6 +879,9 @@ def _thread(handler, timeout, outdir, logfile, fifo_fn, pid_fn,
 
         while True:
             this_timeout = int((timeout_time - time.time()) * 1000)
+            if timeout_extended:
```

Comment:
> For coverage testing, this can reduce the time spent

then it should be:

```suggestion
            if timeout_extended and harness.capture_coverage:
```
otherwise it will disable extended timeout for non-coverage 'late' output as well

Reply:
Coverage is the primary reason for this change, but non-coverage can also benefit in a minor way. Is it likely that any tests print nothing for over a second after the end of the test and then start printing more info that should be captured?